daily-challenges/2024-11/05/Ducnq.k11.spec.ts

```
@@ -0,0 +1,85 @@
+        // Duyệt qua dãy từ a đến b và tính tổng
+        for (let i = a; i <= b; i++) {
+            sum += i;
+        }
+        
+        result.push(sum);  // Thêm tổng vào kết quả
+    }
+    
+    return result;
+}
+
+console.log(calculateTreasure(["1...5", "2...6", "10...15"])); 
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)